Sources/SwerverCore/Request/HttpRequest.swift

```diff
@@ -0,0 +1,34 @@
+import Foundation
+
+public class HttpRequest {
+    
+    private let method: String
+    private let url: String
+    private let httpVersion: String
+    
+    public init(method: String, url: String, httpVersion: String) {
```

Comment:
You can probably make `httpVersion` an enum. That way, users don't accidentally type in the wrong `httpVersion` when using the `init` function.

Reply:
it goes back to the parser class. right now the parser will parse the 3 most important elements from the header from a request (string). HTTP/1.1(http version), / (url) and the request method. once it gathers all 3 piece, it ll then construct an httprequest object. user won't have to enter anything or even call httprequest class for all of this to happen.